README.md

```diff
@@ -1 +1,54 @@
 # Shorebird (un)installer
+
+**❗️ Note: This project is under heavy development. Things will change frequently and none of the code is ready for production use. We will do our best to keep the documentation up-to-date.**
+
+## Getting Started 🚀
+
+1. 🔑 Request an API Key
+1. 🐦 Install Shorebird
+
+   ```
+   dart pub global activate --source git https://github.com/shorebirdtech/shorebird --git-path packages/shorebird_cli
+   ```
+
+1. 🔐 Login
+
+   ```bash
+   shorebird login
+   ? Please enter your API Key: <API-KEY>
+   ✓ Logging into shorebird.dev (7ms)
+   You are now logged in.
+   ```
+
+1. 📦 Create a new build
+
+   ```bash
+   shorebird build
+   ```
+
+   **❗️Note**: If it's the first time, `shorebird` will download and build the shorebird engine which may take some time. The shorebird engine will be cached for subsequent runs.
+
+1. ☁️ Publish a new release
+
+   ```bash
+   # Publish the default release
+   shorebird publish
+
+   # Publish a specific release
+   shorebird publish <path/to/libapp.so>
```

Comment:
I think this mode is going away.

Reply:
Sounds good, I'll remove it from the docs 😄